lib/ramble/ramble/test/get_file_path.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import os
+
+import pytest
+
+import ramble.workspace
+import ramble.config
+import ramble.software_environments
+from ramble.main import RambleCommand
+
+
+# everything here uses the mock_workspace_path
+pytestmark = pytest.mark.usefixtures(
+    "mutable_config",
+    "mutable_mock_workspace_path",
+)
+
+workspace = RambleCommand("workspace")
+
+
+def test_get_file_path(mock_applications, mock_file_auto_create):
+    test_config = """
+ramble:
+  config:
+    disable_progress_bar: true
```

Comment:
I had to disable `tqdm` here otherwise this unit-test fails when running with python 3.7, not sure why though. But later version of python (or tqdm) does not have this issue.